src/cmds.c

```diff
@@ -1033,8 +1090,9 @@ static void cmd_chaddr(struct userrec *u, int idx, char *par)
 #ifdef TLS
   int use_ssl = 0;
 #endif
-  int telnet_port = 3333, relay_port = 3333;
-  char *handle, *addr, *port, *relay;
+  int i, found = 0, telnet_port = 3333, relay_port = 3333;
```

Comment:
please apply all comments from above to this one too.
maybe it makes sense to have a function to check for 1<=x<=65535 now that it is used 4 times

Reply:
done